content/en/synthetics/guide/testing-a-downloaded-file.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Test a downloaded file
+kind: guide
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+---
+
+Journeys on web applications often involve files download: be it an order confirmation for an ecommerce website, exporting the history of a bank account’s transaction to a pdf or a csv, or downloading certificates from official authorities.
```

Comment:
```suggestion
```

Moving these examples under the second paragraph and removing the intro could make this flow better.

Reply:
Good move 👍